tests/library/playwright.config.ts

```diff
@@ -78,6 +78,14 @@ if (mode === 'service2') {
     }
   };
 }
+if (channel === 'webkit-wsl') {
+  connectOptions = { wsEndpoint: 'ws://localhost:3777/' };
+  webServer = {
+    command: 'wsl -d playwright -- npx playwright run-server --port=3777',
```

Comment:
This would need to be dynamic for the cases where several instances are running.